src/guides/modals.md

```diff
@@ -4,30 +4,32 @@ In this section, you'll learn how to use the modal message component.
 
 ![](https://imgur.com/XUbUhdG.png)
 
-> :warning: **Warning:** Modals are only supported for interaction responses  (like slash commands, buttons, select menus, etc), you can't open a modal from just a message command.
+> ⚠️ **Warning:** Modals are only supported for interaction responses  (like slash commands, buttons, select menus, etc), you can't open a modal from just a message command.
 
 ## Creating a Modal
 
 `$newModal[modalID;title]`
 
 - `modalID` - Used in `$onInteraction[ID]` callback. It works same way as [buttons](./buttons.md) and [select menus](./selectmenu.md).
-- `title` - The text which is displayed on top of a modal.
+- `title` - The text which is displayed on top of a modal. This value must be less than or equal to 45 characters.
 
 ## Adding Input Fields in the Modal
 
 `$addTextInput[textInputID;style;label;(minimumLength;maximumLength;required;value;placeholder)]`
 
-- `textInputID` - ID that is used to retrieve the text input in the field. This value should be unique. _(Used in `$input[textInputID]` for retrieving input)_
-- `style` - The modal style, either `short` or `paragraph`. _(Check below for style preview)_\
+- `textInputID` - ID that is used to retrieve the text input in the field. This value must be unique. _(Used in `$input[textInputID]` for retrieving input)_
```

Comment:
Like this is just my nitpicking, but wouldn't a link to the `$input[]` function be better than what currently is in the "note"?